examples/cifar10/cifar10-sim/README.md

```diff
@@ -0,0 +1,178 @@
+# Simulated Federated Learning with CIFAR-10
+
+This example includes instructions on running [FedAvg](https://arxiv.org/abs/1602.05629), 
+[FedProx](https://arxiv.org/abs/1812.06127), [FedOpt](https://arxiv.org/abs/2003.00295), 
+and [SCAFFOLD](https://arxiv.org/abs/1910.06378) algorithms using NVFlare's FL simulator.
+
+For instructions of how to run CIFAR-10 in real-world deployment settings, 
+see the example on ["Real-world Federated Learning with CIFAR-10"](../cifar10-real-world/README.md).
+
+## (Optional) 1. Set up a virtual environment
+```
+python3 -m pip install --user --upgrade pip
+python3 -m pip install --user virtualenv
+```
+(If needed) make all shell scripts executable using
+```
+find . -name ".sh" -exec chmod +x {} \;
+```
+initialize virtual environment.
+```
+source ./virtualenv/set_env.sh
+```
+install required packages for training
+```
+pip install --upgrade pip
+pip install -r ./virtualenv/min-requirements.txt
+```
+(optional) if you would like to plot the TensorBoard event files as shown below, please also install
+```
+pip install -r ./virtualenv/plot-requirements.txt
+```
+Set `PYTHONPATH` to include custom files of this example:
+```
+export PYTHONPATH=${PWD}/..
+```
+
+### 2. Download the CIFAR-10 dataset 
+To speed up the following experiments, first download the [CIFAR-10](https://www.cs.toronto.edu/~kriz/cifar.html) dataset:
+```
+python3 ../pt/utils/cifar10_download_data.py
+```
```

Comment:
shall we move these two sections to upper level (same for real-world readme)?

Reply:
I'd like to keep the two example instructions standalone.